spec/requests/api/v1/cors_headers_spec.rb

```diff
@@ -2,9 +2,9 @@
 
```

Comment:
Wildcard is not supported everywhere, must use domain for compatibility: https://stackoverflow.com/questions/13146892/cors-access-control-allow-headers-wildcard-being-ignored